scripts/coverage_report.sh

```diff
@@ -65,9 +65,22 @@ if [ $SKIP_PROMPTS -eq 0 ]; then
 fi
 
 # Set the flags necessary for coverage output
-export RUSTFLAGS="-Zprofile -Ccodegen-units=1 -Copt-level=0 -Clink-dead-code -Coverflow-checks=off"
-export RUSTC_BOOTSTRAP=1
-export CARGO_INCREMENTAL=0
+# export RUSTFLAGS="-Zprofile -Ccodegen-units=1 -Copt-level=0 -Clink-dead-code -Coverflow-checks=off"
+# export RUSTC_BOOTSTRAP=1
+# export CARGO_INCREMENTAL=0
+
+# env generated by `cargo llvm-cov show-env` on current CI server
+# ----------------------------------------
+# RUSTFLAGS=" -C instrument-coverage --cfg coverage --cfg trybuild_no_target"
+# LLVM_PROFILE_FILE="/runner/_work/starcoin/starcoin/target/starcoin-%m.profraw"
+# CARGO_INCREMENTAL="0"
+# RUST_TEST_THREADS="1"
+# CARGO_LLVM_COV_TARGET_DIR="/runner/_work/starcoin/starcoin/target"
+# ----------------------------------------
+
+# Set the environment variables needed to get coverage.
+source <(cargo llvm-cov show-env --export-prefix) 
```

Comment:
This only sets the environment variables that cargo-llvm-cov normally sets for cargo, and seems unnecessary for your use case.

Reply:
thanks for this info

Reply:
I cleaned up the script according to your suggestions, and it works fine

Can I improve more in our use case, or could you help approve this change ?

thanks